packages/send/src/json.ts

```diff
@@ -8,15 +8,14 @@ type Res = Pick<S, 'setHeader' | 'end' | 'removeHeader'>
  */
 export const json =
   <Response extends Res = Res>(res: Response) =>
-  (body: any, ...args: any[]): Response => {
+  (body: unknown, cb?: () => void): Response => {
     res.setHeader('Content-Type', 'application/json')
-    if (typeof body === 'object' && body != null) res.end(JSON.stringify(body, null, 2), ...args)
-    else if (typeof body === 'string') res.end(body, ...args)
-    else if (body == null) {
+    if (body == null) {
       res.removeHeader('Content-Length')
       res.removeHeader('Transfer-Encoding')
-      res.end(null, ...args)
-    }
```

Comment:
why are you removing `...args`? having `...args` lets you set additional parameters for res.end